compiler/rustc_data_structures/src/sync.rs

```diff
@@ -244,6 +291,9 @@ cfg_if! {
                 MTLock(self.0.clone())
             }
         }
+
+        pub auto trait DynSend {}
+        pub auto trait DynSync {}
     } else {
         pub use std::marker::Send as Send;
```

Comment:
These exports should be unnecessary also.

Reply:
I used `use DynSend/DynSync as Send/Sync` you adviced before.